content/ADR-153-transform-sdk-component.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: adr
+adr: 153
+title: Transform SDK component
+date: 2022-12-14
+status: Living
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - menduz
+---
+
+## Abstract
+
+This document describes the transform component for the SDK, it is used to spatially place things in the world, including the rotation, scaling, positioning and parenting. The semantics of this component also derive the coordinate system of Decentraland explorers. Transform operations are -as in any of the protocol's CRDT- commutative and idempotent operations.
+
+## Component description
+
+The transform component is used to calculate the world matrix of each entity in the world. To do so, it provides information about the position, scale parent entity and rotation quaternion.
+
+## Serialization
+
+```yaml
+parameters:
+  COMPONENT_ID: 1
+  COMPONENT_NAME: core::Transform
+  CRDT_TYPE: LastWriteWin-Element-Set
+```
+
+The Transform component is serialized using a plain-old C struct due to the amount of times it needs to be serialized and deserialized, it uses a fixed memory layout, represented as `TransformComponent` in the following snippet:
+
+```c++
+struct vec3 {
```

Comment:
Shouldn't this be `Vec3`... trying to be consistent with `TransformComponent`. Same for `cuaternion`

Reply:
done